gossip/src/cluster_info.rs

```diff
@@ -4400,15 +4415,15 @@ mod tests {
 
         let trace = cluster_info44.contact_info_trace();
         info!("cluster:\n{}", trace);
-        assert_eq!(trace.len(), 431);
+        assert_eq!(trace.len(), 452);
```

Comment:
what's the point of verifying this? Do we actually rely on some hard coded size anywhere? Or just that it's less than MTU?

If we care, I would think this should live as a constant with some comment indicating why the size matters.

Sort of outside the scope of this PR, but it caught my eye

Reply:
I think Stephen spent some time because someone broke contact_info_trace so the two lines don't align any more, so he added this test as a reminder: https://github.com/anza-xyz/agave/commit/b3fd87fe8189e7e278d882eb2cacafe1f9503d89

That said, I agree this is brittle protection.